lamindb/curators/core.py

```diff
@@ -5,6 +5,7 @@
 
    Curator
    SlotsCurator
+   AtomicDataFrameCurator
```

Comment:
Why do you prefer the name AtomicDataFrameCurator over SimpleDataFrameCurator? Do you have a precedent that merits the more complicated language? 🤔 

While atomicity is a design principle, it's rarely a naming principle. Or is it?

Reply:
I interpret Atomic as "does one thing only" which is kind of the opposite of the Slot based Curator.

"Simple" is a term that doesn't really convey anything. I don't think that `SimpleDataFrameCurator` has more meaning.